params.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Params holds the parameter mappings specified  by user
+type Params map[PName]PValue
+type PName string
+type PValue string
+
+type ParamFile struct {
```

Comment:
Either we make a file like:

```
key=value
key2=value
....
```

or we call this `TestConfigFile` (or `TestInitFile`), to signal that it is a file that can potentially not only contain params but other configurable options for tests. This would affect the flag name that reads the file too.